package.json

```diff
@@ -50,5 +66,10 @@
       "pre-commit": "npm test",
       "commit-msg": "commitlint -E HUSKY_GIT_PARAMS"
     }
+  },
+  "dependencies": {
+    "@americanexpress/eslint-plugin-one-app": "6.0.0",
```

Comment:
why are these needed?

Reply:
thank you for catching this, i've removed them